decred_wallet/Extensions/Extensions.swift

```diff
@@ -69,3 +69,11 @@ extension Date {
         self = Date(timeIntervalSince1970: TimeInterval(milliseconds) / 1000)
     }
 }
+
+extension UIRefreshControl {
+    func programaticallyBeginRefreshing(in tableView: UITableView) {
+        beginRefreshing()
```

Comment:
`self.beginRefreshing()` please, so one does not begin to wonder where this `beginRefreshing` function is written.